test/parser/expect/section-contexts/titles-no-high-levels-permissive.txt

```diff
@@ -1,11 +1,13 @@
 ((output
   (ok
    (((f.ml (1 0) (1 7))
-     (4
+     (5
       (label
        ((root (f.ml (page f.ml) 00000000000000000000000000000000) f.ml)
         (label foo)))
       (((f.ml (1 3) (1 6)) (word Foo))))))))
  (warnings
   ( "File \"f.ml\", line 1, characters 0-7:\
-   \n'5': bad section level (2-4 allowed)")))
+   \n'6': bad heading level (0-5 allowed)"
+    "File \"f.ml.mld\":\
+   \npage title is required, add one with {0 ...}")))
```

Comment:
This test is only accidentally triggering the check for a missing title (it's not the main purpose of the test). I suggest adding a test specifically for that, with `sections_allowed:`All`, and a valid non-title section inside. Perhaps we also should add a test that just has text, to make sure that it doesn't take parsing a section to notice that there is no title.